app/helpers/application_helper.rb

```diff
@@ -93,17 +93,22 @@ def icon_url(name)
   def cloudinary(url, width = nil, _quality = 80, _format = "jpg")
     return url if Rails.env.development? && (url.blank? || url.exclude?("http"))
```

Comment:
I'm curious about the usage of this line.  It seems like a strange thing to return either a blank string or a non-http url in amongst this stuff?  But maybe it makes it easier to run the dev environment?